### ISSUE TITLE
reCAPTCHA not available

### ISSUE THREAD

**Issue by bodtx** [flathub/com.fightcade.Fightcade#123]
Trying to register but I have an error saying it cannot contact recaptcha
![image](https://user-images.githubusercontent.com/1039550/224261653-f94e839d-3081-41ec-965b-ab9d1e961527.png)

Version installed
```

Fightcade - Play arcade games online.

          ID: com.fightcade.Fightcade
         Ref: app/com.fightcade.Fightcade/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 2.2
     License: LicenseRef-proprietary
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 350,0 MB
     Runtime: org.freedesktop.Platform/x86_64/22.08
         Sdk: org.freedesktop.Sdk/x86_64/22.08

      Commit: 83b38b2e83d090d4c1effdc422f686e99d599aeebc1de9a5c875f378347d718f
      Parent: 8d332362ee14e3c8813cbd009d9ea62d95c007938481fe9d1121f7e844a18d56
     Subject: Update FBNeo training mode workaround (3596a97a)
        Date: 2023-01-26 10:39:48 +0000

```

**Comment by Pobega** [comment]
Yeah, I've noticed this happening to some people on the Discord but I don't know the root cause and it's not something I can reproduce so there isn't much I can do unfortunately.

**Comment by bodtx** [comment]
is it mandatory only for creating an account or also for login?

**Comment by Pobega** [comment]
I believe it's only for account creation - login works fine.

**Comment by zrooda** [comment]
Same issue, reCAPTCHA doesn't seem to load properly and registration is impossible for new players.

**Comment by bodtx** [comment]
@zrooda I had to create account under Windows. 
Then Login is ok with the flat pack

**Comment by ngoomie** [comment]
Yeah I've been having the same issue, I meant to create an issue here some time ago but then I managed to forget lol

I'm using the Flatpak on Fedora 38 for the record

**Comment by marmaladefx** [comment]
If you are on immutable-linux or musl-linux and want to stick to flatpak, the most straightforward solution is to use Boxes (`org.gnome.Boxes`) EZ GUI to install a glibc-linux desktop.

Be warned, the `org.gnome.Platform` runtime, plus dependencies require to boot a glibc Linux desktop app are steep. Set aside at least 5 GiB if you are a normie. By normie I mean you just want to install GNOME or KDE inside a VM and register Fightcade there.

Fightcade devs build off of Ubuntu so the latest LTS an obvious choice. Debian 12 is adjacent to Ubuntu and has better privacy.

**Comment by Soundtoxin** [comment]
Just ran into this as well on SteamOS on my Steam Deck. I was already annoyed I couldn't register from a browser on my PC and that I had to hook up a keyboard to my Deck, but now it doesn't even work. What a mess.
I don't use Discord, so won't be joining one to discuss this. Hopefully the devs check the issues for their flatpak occasionally. This seems like a fairly major issue.

**Comment by abda11ah** [comment]
![Screenshot_20231213_092428](https://github.com/flathub/com.fightcade.Fightcade/assets/1699034/35ec1722-2914-48da-a678-17fe4eefc20b)

I just had the same bug today on Manjaro KDE with fightcade in Flatpak version, it's really annoying, especially since I already have a fightcade account but I can't request a new password.

What's more, I can't create the account on the fightcade website.

**Comment by Soundtoxin** [comment]
What I ended up doing to work around this back then was installing fightcade on another machine just to register, then I was able to log in on the one I actually wanted to play on. Though this negative experience among other issues (I think I wasn't able to move when I loaded up a game's training mode for some reason, also people constantly challenging you to a fight / sending messages while you're just trying to get stuff working is quite stressful) has kept me away from fightcade since.

So there's no password reset process? That sounds pretty bad too.

**Comment by CHJ85** [comment]
This still hasn't been fixed. Can you please fix this?
Otherwise there is no way to use this on Linux.

**Comment by Pobega** [comment]
Sorry, no matter how much you ask me this is outside of my control. I've attempted to debug, opened two upstream bugs with the Electron wrapper I'm using and have asked pof but it still persists. 

You're going to have to ask the Fightcade devs to figure out why this is happening, or to offer an alternative registration method.

**Comment by CHJ85** [comment]
@Pobega I found the flatpak on their website, so I thought it was an officially developed build. Sorry.
But does the lack of a recaptcha have something to do with how the electron app handles that particular script, or is it a wine limitation? If it's the latter, perhaps a better approach would be to avoid Wine and use the Linux builds of these emulators. Or if it's Electron related, perhaps using Flutter instead is an option.

**Comment by Pobega** [comment]
The Frontend doesn't use wine, so it's related to Electron.

I'm not an expert but I don't believe Flutter is an option; the goal of the Flatpak is just to sandbox the already existing Fightcade application, AFAIK Flutter would require a rewrite of the client.

For the record this Flatpak is "officially endorsed", not officially supported - an important distinction. I'm the primary maintainer of the Flatpak but not part of the Fightcade team, just a Linux nerd who likes fighting games.

**Comment by Pobega** [comment]
Should be fixed by #149 , found by a user on Discord. Sorry for the trouble, this one was a little tough to track down.
